packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -331,6 +353,12 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
 
   Matrix4? _lastTransform;
 
+  /// Whether scrolling up/down on a trackpad or Magic Mouse should map to
+  /// zooming in/out. Settings this to true will help in reproducing behaviours
+  /// pre-Flutter 3.3 trackpad gestures rewrite.
+  ///
+  /// Defaults to false.
+  bool trackpadPanShouldActAsZoom;
```

Comment:
Nit: As usual for me, I feel like there is a better name we could use but I don't know what it is...  Maybe `trackpadScrollCausesScale` or `trackpadScrollScales`?  I can think of two possible problems with the current name:

  1. Readers might not know what a "trackpad pan" gesture is.  I think it could possibly be ambiguous as to whether it uses one or two fingers.  Maybe using "scroll" is more specific, because it's the same gesture you would use to scroll a webpage etc.?
  2. I've tried to make all of the naming around zooming/scaling just use the word "scale" for consistency, so I think this one should use it too.

Reply:
`trackpadScrollCausesScale` is fine with me.